README.md

```diff
@@ -64,7 +64,7 @@ Build Status
 Project History
 ---------------
 
-This project began as a [CodeProject article](http://www.codeproject.com/KB/install/dotNetInstaller.aspx) and was maintained on the [DevAge website](http://www.devage.com/) and [SourceForge](https://sourceforge.net/projects/dotnetinstaller) by David Icardi. It continued its journey through [CodePlex](http://dotnetinstaller.codeplex.com) and is now officially maintained on [Github](https://github.com/dblock/dotnetinstaller).
+This project began as a [CodeProject article](http://www.codeproject.com/KB/install/dotNetInstaller.aspx) and was maintained on the [DevAge website](http://www.devage.com/) and [SourceForge](https://sourceforge.net/projects/dotnetinstaller) by [Davide Icardi](https://github.com/davideicardi). It continued its journey through [CodePlex](http://dotnetinstaller.codeplex.com) and is now officially maintained on [Github](https://github.com/dblock/dotnetinstaller).

```

Comment:
@davideicardi is alive!

Reply:
> @davideicardi is alive!

Absolutely :smiley: !
Nice to see that my old projects are alive too, thanks to all contributors!